examples/prerecorded/callback/README.md

```diff
@@ -0,0 +1,31 @@
+# Callback Example
+
+This example shows how to use the [Callback](https://developers.deepgram.com/docs/callback) functionality on the Prerecorded API.
+
+> **_NOTE:_** To use this example, the `endpoint` component must run somewhere with a public-facing IP address. You cannot run this example locally behind your typical firewall.
+
+## Configuration
+
+This example consists of two components:
+- `endpoint`: which is an example of what a callback endpoint would look like. Reminder: this requires running with a public-facing IP address
+- `callback`: which is just a Deepgram client posts a PreRecorded transcription request using an audio file located at the following URL: [https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav](https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav)
+
+The `callback` component requires the Deepgram API Key environment variable to be configured to run.
+
+```sh
+export DEEPGRAM_API_KEY=YOUR-APP-KEY-HERE
+```
+
+## Installation
+
+Run the `endpoint` application using an SSL certificates to a system on the public-facing internet. This could be an EC2 instance on AWS, an instance on GCP, or etc.
+
+On the `callback` project, modify the IP address constant in the code with the public-facing IP address of your EC2, GCP, etc instance.
```

Comment:
Can a user run this locally too? Having another server is an extra lift to get started with this example.

Reply:
They can... they just need a public-facing IP address since the DG platform needs to actually hit your endpoint. You can do this by setting up port-forwarding on your router (assuming your ISP won't block/filter hosting a server).

I think someone who knows anything about networking should know how to do this, but if you have any ideas on what to say to someone who doesn't have that networking experience, I would be open to updates.

Reply:
`ngrok` is the go-to for simple callback testing.

Brief overview: run `ngrok http 8080` which opens up the local host port 8080, and gives a public URL `https://e00a-24-165-89-177.ngrok-free.app/` (for example).  Then in another terminal, run `nc -l 8080` to listen on port 8080 for incoming messages.

The request url is then `https://api.deepgram.com/v1/listen?model=general&callback=https://e00a-24-165-89-177.ngrok-free.app`.  The request immediately gets a 200, and the nc terminal eventually gets the Deepgram callback response JSON printed out.

Not saying it's strictly necessary to incorporate this into your example, but I do think it's worth mentioning somewhere to provide a lightweight local callback option!

Reply:
Sounds good. Will add that in.